portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,67 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Pablo's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+        body {
+            background-image: url('background-portfolio.jpg');
+            background-repeat: no-repeat;
+            background-attachment: fixed;
+            background-size: cover;
+        }
+        img {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            border-radius: 5%;
+        }
+    </style>
   </head>
-  <body>
+  <body> 
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="header"><b>Pablo Dueñas' Portfolio:</b></h3>
+      <img id="mainImage" src="pablo-photos.JPG" alt="Photos of Pablo" style="width:500px;height:500px">
+      <hr>
+      <h3 id="header"><i>About me &#128515;:</i></h3>
```

Comment:
The `<i>` tag also italicizes the emoji. Emojis are generally not italicized (I didn't even know they could be until now).